### PR TITLE
Fix e2e test timeout

### PR DESCRIPTION
**Pull request by laneli**
Protractor waits for async tasks to complete - in case of rxjs timer they do not finish so the tests time out
To tackle this it is necessary to run this kind of tasks in ngZone.runOutsideAngular in order to not block Angular.
https://github.com/angular/protractor/blob/master/docs/timeouts.md#waiting-for-angular

Signed-off-by: Elisabeth Lang <elisabeth.lang@dynatrace.com>

### CODE REVIEW

File: bridge/client/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -15,10 +15,10 @@ export class DashboardComponent implements OnInit, OnDestroy{
   public logoInvertedUrl = environment?.config?.logoInvertedUrl;
   public isQualityGatesOnly: boolean;
 
-  private readonly _projectTimerInterval = 30 * 1000;
+  private readonly _projectTimerInterval = 5 * 1000;
```
Comment (christian-kreuzberger-dtx) [inline bridge/client/app/dashboard/dashboard.component.ts:18]: Is this on purpose? I think 30 seconds was already okay, wasn't it?

Comment (laneli) [reply bridge/client/app/dashboard/dashboard.component.ts]: Yes you are right, this was a leftover from testing

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**